website/content/en/status/report-2023-04-2023-06/core.adoc

```diff
@@ -6,35 +6,35 @@ The FreeBSD Core Team is the governing body of FreeBSD.
 
 ==== DevSummit 202305
 
-The Core Team has presented the status update at the DevSummit 202305.
-Slides are available at link:https://wiki.freebsd.org/DevSummit/202305[]
+The Core Team has presented the status update at the FreeBSD Developer Summit, 17th–18th May.
```

Comment:
Since line 26 still refers to this summit as DevSummit 202305, maybe add it here as well in a parenthesis so that it is clear that it is the same thing:
```suggestion
The Core Team has presented the status update at the FreeBSD Developer Summit, 17th–18th May (DevSummit 202305).
```

Or change it below too.

Reply:
We have: 

- **DevSummit 202305** as the immediately preceding subheading (line 7)
- `DevSummit/202305` within the subsequent sentence. 

Whilst expanding the phrase at the next visible line (9) I intentionally **refrained from repeating the contraction** in the midst of the other two contractions. Visually: 

----

![image](https://github.com/freebsd/freebsd-doc/assets/192271/19c1f5f4-8989-4a4c-adaf-ff4d5b3e07e7)

Reply:
> … Or change it below too.

The contraction begins the report, so I think the contraction is OK at line 26. 

It's not so far away that readers might lose sight of its meaning: 

----

![image](https://github.com/freebsd/freebsd-doc/assets/192271/b7131343-40e6-4991-9705-793580b9c448)